src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -231,9 +231,9 @@
     </fields>
     <fields>
         <fullName>InstallmentFrequency__c</fullName>
-        <description>The Installment Frequency along with Installment Period defines the installment schedule of this Recurring Donation. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule would be defined as &quot;every 3 months&quot;.</description>
+        <description>The Installment Frequency along with Installment Period defines the installment schedule of this Recurring Donation. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule is defined as &quot;every 3 months&quot;.</description>
         <externalId>false</externalId>
-        <inlineHelpText>The Installment Frequency along with Installment Period defines the installment schedule of this Recurring Donation. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule would be defined as &quot;every 3 months&quot;.</inlineHelpText>
+        <inlineHelpText>The Installment Frequency along with Installment Period defines the installment schedule of this Recurring Donation. For example, if Installment Period is Monthly and Installment Frequency is 3, the schedule is defined as &quot;every 3 months&quot;.</inlineHelpText>
```

Comment:
@sam-knox Since this field was already deployed to all existing customers, this change will not be applied to all the existing orgs. To allow everyone to see this, the field (including the new help text) will need to be added to the `unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object` file.

Reply:
The same for the StartDate field

Reply:
@force2b - I just added both to the unmanaged file. Is it ok for it to be in both places?